### PR TITLE
Bump version to v1.2.0-rc3

### PR DESCRIPTION
**Pull request by dongluochen**
Signed-off-by: Dong Chen <dongluo.chen@docker.com>

### CODE REVIEW

**Comment by vieux** [comment]
not sure `Fix docker host restart in integration tests` is needed but LGTM anyway

File: CHANGELOG.md [hunk 1]
```diff
@@ -32,6 +35,14 @@
...
 - Force to validate min and max refresh interval to be positive
 - Skip unstable tests from Docker bug 14203
+- Fix race condition between node removal from discovery and scheduler
+- Fix data race with node failureCount
+- Display warning message if an engine has labels with "node=xxx"
+
+#### Discovery
+
+- Remove parameter which is not used in createDiscovery
+- Fix Consul Leader Election Failure on multi-server
```
Comment (nishanttotla) [inline CHANGELOG.md]: nit: `Failure` -> `failure`

**Comment by nishanttotla** [comment]
I personally think `Fix docker host restart in integration tests` shouldn't be part of the changelog.

Otherwise LGTM.

File: CHANGELOG.md (hunk 1 above)
Comment (dongluochen) [inline CHANGELOG.md]: fixed.

**Comment by dongluochen** [comment]
Updated. Thanks for quick review!

**Pull request merged**